docs/UserGuide.md

```diff
@@ -548,7 +652,7 @@ For advanced users, feel free to update patient and appointment data directly by
 
 <br>
 
-## FAQ
+## **FAQ**
 
 **Q**: How is my data stored?<br>
 **A**: Your data is stored in [.json](https://www.json.org/json-en.html) files, located in the `data` folder. The details of patients are stored in the `addressbook.json` file, while the appointments are stored in the `appointmentbook.json` file.
```

Comment:
For the images, can we update the screenshots to match the default ones with the correct fields? Should we 
 also make them consistent in terms of the number of data and size of screenshot?

<img width="472" alt="Screenshot 2024-11-07 at 06 57 52" src="https://github.com/user-attachments/assets/79e226ed-b3bc-4c0f-9e90-02238ee4084f">

<img width="516" alt="Screenshot 2024-11-07 at 06 59 29" src="https://github.com/user-attachments/assets/60487b19-3f33-469f-a3d1-753e5d4eee04">

Reply:
Would it be better to just have it as a code excerpt instead of a screenshot?